coding-concepts.js

```diff
@@ -28,8 +32,10 @@ const greeting = "Hello World!"
 // console.log(greeting[4])
 
 // a) Your answer:
-// b) Verify and explain:
+//I believe that the output of this code will be "Hello World" 4 times. Greeting is the variable that was created to invoke "Hello World" and the 4 is the number of times we would like to invoke the variable.
 
+// b) Verify and explain:
+// The output was o. I believe what happen is that the code calculated the index of the string. Thus, by writing 4, the output will be the fourth location of the index which was o.
```

Comment:
Yes, the `[4]` signifies the 4th index of the assigned value to the greeting variable. Because it is a string `Hello World!` the 4th index will be the character `o`.